index.html

```diff
@@ -0,0 +1,134 @@
+                    <li class="works-heading">Tonic</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <p class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <button>See project</button>
+                </ul>
+            </li>
+            <li class="container2">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot4.png" alt=""></li>
+                    <li class="works-heading">Multi-Post Stories</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <p class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <button>See project</button>
+                </ul>
+            </li>
+        </ul>
+    </section>
+    <section id="about-self">
+        <h3 class="aboutself-heading">About myself</h3>
+        <p class="aboutself-paragragh">Hello I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see  and have a project you need coded, don't hesitate to contact me.</p>
+        <h4 class="connect2">LET'S CONNECT</h4>
+            <ul id="social-icons">
+                <li><a href="#" class="icon"><img src="Assets/files/assets/github.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/Linkedin icon.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/Vector.jpg" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/twitter.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/medium.png" alt="Loading"></a></li>
+            </ul>
+        <button class="resume">Get my resume</button>
+
+        <ul class="langues">
+            <li class="languages1">Languages</li>
+            <div class="arrow"> <img src="Assets/files/arrow.png" alt=""></div>
+                <ul id="tech">
+                    <li><button class="tag tag1"> Javascript</button></li>
+                    <li><button class="tag tag2"> HTML</button></li>
+                    <li><button class="tag tag3"> CSS</button></li>    
+                </ul>
```

Comment:
- [ ]  As the previous reviewer pointed out,  only **li** tags should be direct children of the **ul** tag 😊 
The issue here is that you are having `<div class="arrow"> <img src="Assets/files/arrow.png" alt=""></div>` and 
```
<ul id="tech">
  <li><button class="tag tag1"> Javascript</button></li>
  <li><button class="tag tag2"> HTML</button></li>
  <li><button class="tag tag3"> CSS</button></li>
</ul>
```
 as a direct children of  `<ul class="langues">`

You might fix the structure of your code this way:

```
<li class="languages1">
	<h3>Languages <img src="Assets/files/arrow.png" alt="Arrow icon"></h3>
  <ul id="tech">
     <li><button class="tag tag1"> Javascript</button></li>
     <li><button class="tag tag2"> HTML</button></li>
     <li><button class="tag tag3"> CSS</button></li>    
  </ul>
</li>
```